code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -769,6 +769,8 @@
 		if(method == REAGENT_INGEST || (method == REAGENT_TOUCH && prob(25)))
 			if(M.stat == DEAD)
```

Comment:
Should this possibly bring the cling back to life if they're in stasis (TRAIT_FAKE_DEATH I think)?

Reply:
No, that was the original pitch but it got declined